### ISSUE TITLE
Flame Scans: HTTP Error 403

### ISSUE THREAD

**Issue by DatHypnoboi** [tachiyomiorg/tachiyomi-extensions#8178]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

0.11.1

### Android version

11

### Source information

Flame Scans 1.2.15

### Steps to reproduce

1. Open Flame Scans source
2. Open any manga/manwha/manhua

### Expected behavior

Manga info should show

### Actual behavior

Gives "Http error 403" message instead

**Comment by mcOz1** [comment]
I have the same problem. When I try to open a page in browser on application I got 'access denied', but when I try without application, for example in chrome, everything working normally.
![1](https://user-images.githubusercontent.com/71328023/126330860-008844ec-a5c8-492c-8ae7-2537b331dd7e.PNG)
![2](https://user-images.githubusercontent.com/71328023/126330861-d2cfeb0d-cebd-489a-91fc-556c7708feb1.PNG)

**Comment by Seishirou101** [comment]
They block tachi at peak times/when they want. migrate or wait for them to unblock

**Comment by 8227846265** [comment]
Same time is happening from my end on Flame Scans extension.

**Comment by hyoretsu** [comment]
> They block tachi at peak times/when they want. migrate or wait for them to unblock
Last update I received was from at most 3 weeks ago, so damn
